qt/python/mantidqt/mantidqt/widgets/colorbar/colorbar.py

```diff
@@ -384,3 +415,15 @@ def _enable_normalisation_option(self, option_index: int) -> None:
         if not self.norm.model().item(option_index, 0).isEnabled():
             self.norm.model().item(option_index, 0).setEnabled(True)
             self.norm.setItemData(option_index, "", Qt.ToolTipRole)
+
+    def _set_cmax_box_outline(self):
+        if not self.cmax.hasAcceptableInput():
+            self.cmax.setStyleSheet("border: 1px solid red")
+        else:
+            self.cmax.setStyleSheet(self.cmax_default_style_sheet)
+
+    def _set_cmin_box_outline(self):
+        if not self.cmin.hasAcceptableInput():
+            self.cmin.setStyleSheet("border: 1px solid red")
+        else:
+            self.cmin.setStyleSheet(self.cmin_default_style_sheet)
```

Comment:
These look like they could take `cmax`/`cmin` as a parameter and could be one function (assuming the stylesheets don't vary too much between max and min?)